tests/testing/asserters_test.py

```diff
@@ -889,6 +920,28 @@ def test_assert_equal(self, size, left_as_arkouda, right_as_arkouda):
                 assert_equal(df, df2)
             assert_equivalent(convert_left(df), convert_right(df2))
 
+    @pytest.mark.skip_if_max_rank_less_than(2)
+    @pytest.mark.parametrize("size", pytest.prob_size)
```

Comment:
I think this need to be `@pytest.mark.skip_if_max_rank_less_than(3)` for the same reason about `shape=(2, 2, size)`

Reply:
Sorry about that, it should be fixed now.

Reply:
no worries!